jsettlers.mapcreator/src/jsettlers/mapcreator/localization/labels_de.properties

```diff
@@ -12,100 +13,184 @@
 # LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 # DEALINGS IN THE SOFTWARE.
 ###############################################################################
-templatedescr   = Vorlage %s
-movabledescr    = %s platzieren
-stackdescr      = %d %s platzieren
-commondescr     = %s platzieren
-treedescr       = Baum platzieren
-stonedescr      = %d Steine platzieren
-buildingdescr   = Gebäude %s platzieren
-deletedescr     = Objekte löschen
-landscapedescr  = %s
-changeheightdescr   = Höhen ändern
-increaseheightdescr = Erhöhen
-decreaseheightdescr = Höhe veringern
-flatten             = Flach machen
-fixheights          = Höhen-/Resourcenprobleme beheben
-setstartpoint       = Startpunkt setzen
-place_stacks        = %s
-place_resource      = %s
-remove_resource      = Resourcen löschen
-
-fuzzy_line          = Abgefl. L.
-line                = Linie
-circle_line         = Kreisf. L.
-grid_circle         = Ausged. K.
-grid_line           = Ausged. L.
-noisy_circle_line   = Rauschlinie
-points              = Punkte
-line_inner          = Innerer Bereich (%)
-
-save = Speichern
-undo = Rückgängig
-redo = Wiederholen
-errors          = Fehler
-errors_n        = %d Fehler
-
-toolbox = Werkzege
-category-landscape = Landscahft
-category-heigths = Höhen
-category-land-resources = Resourcen
-category-objects = Objekte
-category-settlers = Siedler
-category-worker = Arbeiter
-category-specialist = Spezialisten
-category-soldier = Krieger
-category-materials = Materialien
-category-mat-build = Bauen
-category-mat-food = Essen
-category-mat-resources = Rohstoffe
-category-mat-tools = Werkzeug
-category-mat-weapons = Waffen
-category-buildings = Gebäude
-category-resources = Rohstoffverarbeitung
-category-food = Essen
-category-military = Militär
-category-social = Sozial
+#
+# ** Tips to translate **
+# May use http://essiembre.github.io/eclipse-rbe/ as editor, makes it a lot easyer
+# Use a dot (.) to group tags by Category, this is common in property files, and then
+# it's displayed as groups in the Translator plugin mentioned above
+
+about.developer  = Entwickler
+about.header     = \u00DCber
```

Comment:
Can you change this to UTF8?
I know it is not properties default, but we read all files as UTF8 to not have encoding issues.